Shared.Tests/Asset/ParametersTest.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
```

Comment:
This is actually Admin/Upload API parameters test, and not asset, please move it to a separate folder